src/Psalm/Internal/PhpVisitor/Reflector/ClassLikeNodeScanner.php

```diff
@@ -325,17 +337,6 @@ public function start(PhpParser\Node\Stmt\ClassLike $node): ?bool
                     $this->file_storage->has_visitor_issues = true;
                     $storage->has_visitor_issues = true;
                 }
-                $storage->class_implements['backedenum'] = 'BackedEnum';
```

Comment:
This means we no longer consider enums backed with an invalid type as actually backed.